website/docs/GettingStarted.md

```diff
@@ -32,44 +32,58 @@ If you're in the root directory of an existing project, run:
     {label: 'NPM', value: 'npm'},
     {label: 'Yarn', value: 'yarn'},
     {label: 'pnpm', value: 'pnpm'},
+    {label: 'bun', value: 'bun'},
   ]
 }>
 <TabItem value="npm">
 
 ```sh
-npm init wdio .
+npm init wdio@latest .
 ```
 
 or if you want to create a new project:
 
 ```sh
-npm init wdio ./path/to/new/project
+npm init wdio@latest ./path/to/new/project
 ```
 
 </TabItem>
 <TabItem value="yarn">
 
 ```sh
-yarn create wdio .
+yarn create wdio@latest .
```

Comment:
This will actually fail with 

```
/bin/sh: /Users/christian.bromann/.yarn/bin/create-wdio@latest: No such file or directory
error Command failed.
Exit code: 127
Command: /Users/christian.bromann/.yarn/bin/create-wdio@latest
Arguments: ./sss
Directory: /private/tmp
Output:
```

`yarn create wdio .` works

Reply:
Oh that's odd! Good to know, will test with yarn, pnpm and bun 👍

Reply:
Yarn seems to be an exception to the rule here, good catch!